app/gui/src/project-view/components/GraphEditor/widgets/WidgetSelection/styles.ts

```diff
@@ -68,7 +68,7 @@ function middleware(isTopLevel: boolean, limitWidth: boolean, rootElement: Ref<O
 export function activityDropdownStyles(
   floatReference: Ref<Opt<HTMLElement>>,
   dropdownElement: Ref<Opt<HTMLElement>>,
-  rootElement: Ref<HTMLElement | undefined>,
+  rootElement: Ref<HTMLElement | null | undefined>,
```

Comment:
Can be replaced with `Opt<HTMLElement>`, as above. I have no idea why it was not updated before.